lib/admin_presets_export.class.php

```diff
@@ -87,7 +87,7 @@ public function execute() {
             $this->id = $preset->id;
 
             // We must ensure that there are settings selected.
-            foreach (filter_input_array(INPUT_POST) as $varname => $value) {
+            foreach ($_POST as $varname => $value) {
```

Comment:
Direct access to POST value is not allowed to prevent "XSS and other security issues".

Reply:
Remember to fix this later @kecherou